jetty-embedded-12-1-ee11/src/main/java/org/jboss/arquillian/container/jetty/embedded_12_ee11/ArquillianAppProvider.java

```diff
@@ -223,18 +197,17 @@ public ContextHandler createContextHandler(final App app) throws Exception {
             webAppContext.setAttribute(WebAppContext.TEMP_DIR, config.getTempDirectory());
         }
 
-        webAppContextProcessors.forEach(processor -> processor.process(webAppContext, ((ArchiveApp)app).archive));
+        webAppContextProcessors.forEach(processor -> processor.process(webAppContext, archive));
 
         return webAppContext;
     }
 
-    @Override
-    public String getEnvironmentName() {
-        return "ee11";
-    }
-
-    @Override
-    public void setDeploymentManager(DeploymentManager deploymentManager) {
-        this.deploymentManager = deploymentManager;
+    private static boolean isWebArchiveFile(Path path) {
+        if (!path.toFile().isFile()) {
+            return false;
+        } else {
+            String name = path.toFile().getName().toLowerCase(Locale.ENGLISH);
+            return name.endsWith(".war") || name.endsWith(".jar");
+        }
     }
```

Comment:
This can be replaced with ...
```suggestion
    private static boolean isWebArchiveFile(Path path) {
        return FileID.isExtension(path, "war", "jar");
    }
```

That performs both the isFile check, along with properly getting the extension.

Reply:
fixed 5a1b7e6f828468d30ee27c918e6692c92272c3d9